src/main.rs

```diff
@@ -66,24 +69,25 @@ fn main() {
         list_fuzz_targets()
             .map(|_| ())
     } else {
-        println!("Invalid arguments. Usage:\n{}", USAGE);
+        utils::write_to_stderr(format!("Invalid arguments. Usage:\n{}", USAGE).as_str());
```

Comment:
Actually, the usage string should not be bold, so you should use a different call to do this.